### DISCUSSION TITLE
Push Master to PyPI

### DISCUSSION THREAD

**Discussion by CuboidRaptor** [Pycord-Development/pycord#535] [Q&A] [answered]
Would it be possible to allow slash commands and stuff on the PyPI version? My pip installation cannot handle git for some reason and I plan to send this to Heroku, which would make everything painful if I also have to figure out how to install git on Heroku.

**Comment by esmaycat** [comment]
I don't think that 2.0 is done yet so there is no point releasing it...

**Comment by Lulalaby** [comment, marked as answer]
No we won't push it till the 2.0 release, which is soon. Sorry.
